integration-tests/utils/quay/quay-deployment-with-odf.sh

```
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+SHARED_DIR=$(mktemp -d)
+#Get the credentials and Email of new Quay User
+QUAY_USERNAME=admin
+QUAY_PASSWORD=<password>
```

Shellcheck (0.11.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)